### ISSUE TITLE
7.0 to 8.0 migrate error. "Wrong value for product.template.type: 'product'"

### ISSUE THREAD

**Issue by cewolf2002** [OCA/OpenUpgrade#442]
Please Help.
We tried to migrate our 7.0 database to 8.0.
But always stop here.
I tried to compare database between this one and another 7.0 database but no difference.

Server running ubuntu 12.04.5
All modules are already up-to-date.
And openerp's version is 20160222

Full log:
http://pastebin.com/mj8bUk8d

**Comment by hbrunn** [comment]
this looks like you have some module installed that changes the selection of the `type` field on `product.template`. To have this survive the migration, duplicate the column before migration, normalize all values there to one one the two the product module expects (https://github.com/OCA/OCB/blob/8.0/addons/product/product.py#L533), and write back the other values after migration.

**Comment by cewolf2002** [comment]
Thanks for response.
We have 6714 products now.
I checked column type. 
type        count
product 6712
service 2

How should I fix it?

**Comment by cewolf2002** [comment]
I tried to install database_cleanup.
Then I passed this part after purging all.
But now, another error message stopped me.

2016-02-24 11:15:51,846 5565 WARNING Hanyu_migrated OpenUpgrade: Module stock, table stock_warehouse, column view_location_id: Creating new view location for warehouse WH2
2016-02-24 11:16:02,636 5565 ERROR Hanyu_migrated OpenUpgrade: stock: error in migration script /var/tmp/openupgrade/8.0/addons/stock/migrations/8.0.1.1/post-migration.py: list index out of range
2016-02-24 11:16:02,636 5565 ERROR Hanyu_migrated OpenUpgrade: list index out of range

Full log:
http://pastebin.com/iu2xtcmw
